src/database/migrations/20241217110259-update-reports-data.js

```diff
@@ -0,0 +1,293 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		// Insert the report data into the reports table
+		await queryInterface.bulkInsert('reports', [
+			{
+				code: 'total_number_of_sessions_attended',
+				title: 'Total number of sessions attended',
+				description: 'Total sessions attended by user in big number',
+				report_type_id: 1,
+				created_at: Sequelize.literal('CURRENT_TIMESTAMP'),
+				updated_at: Sequelize.literal('CURRENT_TIMESTAMP'),
+				config: '{}',
+				organization_id: 1,
+			},
+			{
+				code: 'total_hours_of_learning',
+				title: 'Total hours of learning',
+				description: 'Total hours of learning by user in big number',
+				report_type_id: 1,
+				created_at: Sequelize.literal('CURRENT_TIMESTAMP'),
+				updated_at: Sequelize.literal('CURRENT_TIMESTAMP'),
+				config: '{}',
+				organization_id: 1,
+			},
+			{
+				code: 'split_of_sessions_enrolled_and_attended_by_user',
+				title: 'Total sessions enrolled vs total sessions attended',
+				description: 'Split of sessions enrolled and attended by user in bar chart',
+				report_type_id: 4,
+				created_at: Sequelize.literal('CURRENT_TIMESTAMP'),
+				updated_at: Sequelize.literal('CURRENT_TIMESTAMP'),
+				config: '{}',
+				organization_id: 1,
+			},
+			{
+				code: 'mentee_session_details',
+				title: 'Session details',
+				description: 'Mentee session details table with pagination and downloadable pdf',
+				report_type_id: 3,
+				created_at: Sequelize.literal('CURRENT_TIMESTAMP'),
+				updated_at: Sequelize.literal('CURRENT_TIMESTAMP'),
+				config: JSON.stringify({
+					columns: [
```

Comment:
@sumanvpacewisdom please check with sansriti or product team, which the columns should be filterable.
I think vishwa has this list, discuss with him once